### ISSUE TITLE
[DOC] Widget Development: join Getting Started sections

### ISSUE THREAD

**Issue by ajdapretnar** [biolab/orange3#5122]
Getting Started in Widget Development documentation has two separate sections. Make it one section.
https://orange3.readthedocs.io/projects/orange-development/en/latest/

**Comment by markotoplak** [comment]
Hmm, which docs are we supposed to show on that page anyway? 

The docs you pointed out are from the orange3 repo, but similar docs are also in orange-widget-base. One of these should be deleted. @ales-erjavec, if I understand correctly, orange-widget-base are being updated, so we could delete the ones in orange3?

**Comment by ajdapretnar** [comment]
Yes, this was another issue - overlapping documentation. Ideally, we would have it in one place. @irgolic also agreed that having them in orange-widget-base would make more sense. If docs on the website could point there, it would be great.

**Comment by ales-erjavec** [comment]
> if I understand correctly, orange-widget-base are being updated, so we could delete the ones in orange3?

I guess.

**Comment by irgolic** [comment]
Only, it'd be nice to mention that when writing a new widget, OWWidget should be extended, not OWBaseWidget. I suppose this is mitigated by the example addon.
